### ISSUE TITLE
Test: All links must have a "title" attribute

### ISSUE THREAD

**Issue by jssh** [quailjs/quail#363]
I do not think this is correct. The "title" is supposed to provide additional / advisory information (expand on the meaning of the link). Do not add link titles to all links: if it is obvious from the link anchor and its surrounding context where the link will lead, then a link title will reduce usability by being one more thing users have to look at. •don’t duplicate it with link text (this hurts usability: for example some blind users will hear the same text twice);
•don’t put too much weight on the title attributes as not all screen readers may render it (make sure either surrounding text or anchor text explains the link at least the first time you use it).

**Comment by jessebeach** [comment]
Yes, I agree. This test needs to be refactored, possibly even removed.
